os/drivers/lwnl/lwnl_evt_queue.c

```diff
@@ -61,20 +87,22 @@ struct lwnl_queue {
 static struct lwnl_queue g_queue[LWNL_NPOLLWAITERS];
 static int g_connected = 0;
 
+static int g_totalevt = 0; // debugging
```

Comment:
Like a comment, this is only for debug, so at least, config regarding debug level should be added here.